cibuild.sh

```diff
@@ -51,31 +54,30 @@ do
         BUILD_CONFIGURATION=Release
         shift 1
         ;;
-        --nocache)
-        USE_CACHE=false
+        --cleanrun)
+        CLEAN_RUN=true
         shift 1
         ;;
         --skiptests)
         SKIP_TESTS=true
         shift 1
         ;;
-        --skipcrossgen)
-        SKIP_CROSSGEN=true
-        shift 1
-        ;;
         --skipcommitprinting)
         SKIP_COMMIT_PRINTING=true
         shift 1
         ;;
+        --nocache)
+        # Temporarily ignore this argument until the change to netci.groovy gets merged.
+        # A follow-up PR will be made to remove this ignore.
+        shift 1
+        ;;
```

Comment:
@jaredpar As discussed, this needs to be in this PR until the netci.groovy change (also in this PR) is merged. A follow-up PR removing this will be made at that point.